srcpkgs/slmenu/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'slmenu'
+pkgname=slmenu
+version=0.1
+revision=1
+_git_commit=7e74fa5db73e
+_author=rafaelgg
```

Comment:
No need to use a custom variable just for that.

Reply:
I adapted the template from xss-lock, there is no version, nor releases in the repo.

Reply:
I mean this custom variable: `_author=rafaelgg`

Reply:
get rid of _author and 
distfiles="${homepage}/get/${_git_commit}.tar.gz"   ?

Reply:
````
wrksrc="rafaelgg-${pkgname}-${_git_commit}"
distfiles="https://bitbucket.org/rafaelgg/${pkgname}/get/${_git_commit}.tar.gz"
````

Reply:
ok, got it. Thank you for your patience.